.scripts/deploy-kinsta.sh

```diff
@@ -105,6 +105,9 @@
     if [ 0 != $? ]; then
         # Create a new orphan branch to track deployments
         git checkout --orphan "deploy/${DEPLOY_ENV}"
+
+      # Everything is being tracked, so remove it
```

Comment:
@csrui this is especially of relevance to you.

I found that the `git checkout --orphan` command makes all the files automatically tracked, so it was not following the `.gitignore` rules set by the file that is renamed in the following steps, right after the **rsync**.

This should only be done on the occasion where the branch `deploy/${DEPLOY_ENV}` is being created from scratch. All subsequent updates use the basic `git checkout` command and don't suffer from the same issues.